core/miner.go

```diff
@@ -91,8 +94,8 @@ func (miner *Miner) update() {
 	}
 }
 
-func (miner *Miner) Start(coinbase common.Address) {
-	miner.startCh <- coinbase
+func (miner *Miner) Start(primaryCoinbase, secondaryCoinbase common.Address) {
```

Comment:
Can we just delete this

Reply:
its used right now to set the coinbase, so we have to wait to delete this until we are good

Reply:
deleting this